easybuild/easyconfigs/m/MoviePy/MoviePy-1.0.3-foss-2021b.eb

```diff
@@ -0,0 +1,47 @@
+# This easyconfig was created by Simon Branford of the BEAR Software team at the University of Birmingham.
+# Updated to foss 2021b: J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'PythonBundle'
+
+name = 'MoviePy'
+version = '1.0.3'
+
+homepage = "https://zulko.github.io/moviepy/"
+description = """MoviePy (full documentation) is a Python library for video editing: cutting, concatenations,
+ title insertions, video compositing (a.k.a. non-linear editing), video processing, and creation of custom effects."""
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+
+dependencies = [
+    ('FFmpeg', '4.3.2'),
+    ('Python', '3.9.6'),
+    ('Pillow', '8.3.2'),
+    ('SciPy-bundle', '2021.10'),
+    ('tqdm', '4.62.3'),
+    ('imageio', '2.13.5'),
+]
+
+sanity_pip_check = True
+use_pip = True
+
+exts_list = [
+    ('imageio-ffmpeg', '0.4.7', {
+        'checksums': ['7a08838f97f363e37ca41821b864fd3fdc99ab1fe2421040c78eb5f56a9e723e'],
+    }),
+    ('proglog', '0.1.10', {
+        'checksums': ['658c28c9c82e4caeb2f25f488fff9ceace22f8d69b15d0c1c86d64275e4ddab4'],
+    }),
+    (name, version, {
+        'source_tmpl': 'moviepy-%(version)s.tar.gz',
+        'patches': ['moviepy-1.0.3-setup.py.patch'],
+        'checksums': [('2884e35d1788077db3ff89e763c5ba7bfddbd7ae9108c9bc809e7ba58fa433f5'),
+                      ('beca82ff8dfbc9cd0b97296f310de3ddfc43e2e49344a32375d4438485c3f3df')],
+    }),
+]
+
+sanity_check_paths = {
```

Comment:
This path is already the default, see https://github.com/easybuilders/easybuild-easyblocks/blob/839cdea71872022ec9d6917870aeecf4d271eb48/easybuild/easyblocks/generic/pythonbundle.py#L158

So, I propose to remove it again (sorry... :))